assignments/objects.js

```diff
@@ -19,27 +19,87 @@ const example = {
 
 // Write your intern objects here:
 
+const interns = [{
+  "id": 1,
+  "email": "mmelloy0@psu.edu",
+  "name": "Mitzi",
+  "gender": "F",
+},
+
+{
+  "id": 2,
+  "email": "kdiben1@tinypic.com",
+  "name": "Kennan",
+  "gender": "M",
+  "speak": function(){
+    console.log("Hello, my name is Kennan!");
+  },
+},
+
+{
+  "id": 3,
+  "email": "kmummery2@wikimedia.org",
+  "name": "Keven",
+  "gender": "M",
+},
+
+{
+  "id": 4,
+  "email": "gmartinson3@illinois.edu",
+  "name": "Gannie",
+  "gender": "M",
+},
+
+{
+  "id": 5,
+  "email": "adaine5@samsung.com",
+  "name": "Antonietta",
+  "gender": "F",
+  "multiply": function(x,y){
+    return x*y;
+  }
+}];
+
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(interns[0]["name"]);
+
+// console.log(interns[0].name());
+
+// TypeError: interns[0].name is not a function
+
 // Kennan's ID
 
+console.log(interns[1]["id"]);
+
 // Keven's email
 
+console.log(interns[2]["email"]);
+
 // Gannie's name
 
+console.log(interns[3]["name"]);
+
 // Antonietta's Gender
 
+console.log(interns[4]["gender"]);
+
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+console.log(interns[1]["speak"]())
+
```

Comment:
Instead of adding it directly to the kennan object you can do
```suggestion
intern[1].speak = function() {
    return `Hello, my name is ${this.name}`
}
```

Reply:
If you keep the original code you have to be interns[1].speak() to invoke it